styles/blocks/service/service-item.css

```diff
@@ -0,0 +1,63 @@
+.service-item + .service-item {
+    margin-left: 4%;
+}
+
+.service-item {
+    display: flex;
+}
+
+.service-item__icon {
+    width: 64px;
+    height: 64px;
+
+    flex-shrink: 0;
+}
+
+.service-item__flag-img {
+    width: 50%;
+    height: 71%;
+    margin: 16% 0 0 30%;
```

Comment:
Say goodbye to pixel perfect=)

Reply:
Probably. In the ideal world you’d want to have those images of the same size. If you want, you can try resizing, otherwise stick to setting the sizes.